app/services/flood_risk_engine/address_services/search_service_failure.rb

```diff
@@ -0,0 +1,22 @@
+module FloodRiskEngine
+  module AddressServices
+
+    class SearchServiceFailure
+
+      attr_accessor :exception
+
+      def initialize(ex)
+        @exception = ex
+      end
```

Comment:
there was a previous comment about exceptions being caught and then hard to track so this is to give devs an opportunity to grab/review actual errors at external service level